packages/core/test/unit/checker/checker-facade.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { CheckStatus } from '@stryker-mutator/api/check';
+import { factory } from '@stryker-mutator/test-helpers';
+
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import { CheckerFacade } from '../../../src/checker/checker-facade.js';
+import { CheckerResource } from '../../../src/checker/index.js';
+
+import { createMutantRunPlan } from '../../helpers/producers.js';
+
+describe(CheckerFacade.name, () => {
```

Comment:
I see you've got tests for `check()` and `group()` in here, could you separate them out into their own inner `describe`s?